### PR TITLE
fsql 0.1.0 (new formula)

### PR DESCRIPTION
**Pull request by kashav**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/fsql.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+class Fsql < Formula
+  desc "Search through your filesystem with SQL-esque queries."
+  homepage "https://github.com/kshvmdn/fsql"
+  url "https://github.com/kshvmdn/fsql/archive/v0.1.0.tar.gz"
+  sha256 "7b27588e03127055b40f2a2ef4f554e86684bbdd1a4ef45cdad53f48ec9047e4"
+
+  depends_on "go" => :build
+
+  def install
```
Comment (ilovezfs) [inline Formula/fsql.rb]:
```
    ENV["GOPATH"] = buildpath
    (buildpath/"src/github.com/kshvmdn").mkpath
    ln_s buildpath, buildpath/"src/github.com/kshvmdn/fsql"
    system "go", "build", "-o", bin/"fsql"
```

File: Formula/fsql.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    mkdir_p buildpath/"src/github.com/kshvmdn"
+    ln_s buildpath, buildpath/"src/github.com/kshvmdn/fsql"
+
+    ENV["GOPATH"] = buildpath
+    system "go", "build", "-o", "fsql"
+    bin.install "fsql"
+  end
+
+  test do
+    output = shell_output(bin/"fsql -version")
```
Comment (ilovezfs) [inline Formula/fsql.rb]: This needs to be a functional test.

**Review by ilovezfs — CHANGES_REQUESTED** [review]

File: Formula/fsql.rb (hunk 2 above)
Comment (kshvmdn) [inline Formula/fsql.rb]: Needed to draft a new release, going to open a new PR.

Comment (ilovezfs) [inline Formula/fsql.rb]: OK, thanks!

**Pull request closed without merging**